apps/web/src/views/AgentPlaygroundView.tsx

```diff
@@ -0,0 +1,34 @@
+import { InputMessage } from "~core/components/InputMessage"
+import { useAgentManager } from "~core/providers/agent-manager"
+import { AgentHiringCard } from "~features/agent/AgentHiringCard"
+import { CaptainMonitor } from "~features/agent/CaptainMonitor"
+import { CrewMonitor } from "~features/agent/CrewMonitor"
+
+// Create a screen writing mobile application with a novel UX that no one has seen before, ensure the design is great
+export function AgentPlaygroundView() {
```

Comment:
Oddly, this component shows the web terminal to the right for me:
<img width="1694" alt="Screenshot 2023-05-07 at 2 35 28 AM" src="https://user-images.githubusercontent.com/1011391/236666959-a4f9dfe1-83a0-4d1b-9b9a-d372325ae99f.png">

Reply:
ah because it's inside a webvmprovider?

Reply:
Yup - the agent can leverage it headlessly!